server/schedule/placement/fit.go

```diff
@@ -52,6 +52,46 @@ func (f *RegionFit) IsCached() bool {
 	return f.mu.cached
 }
 
+// Replace return true if the replacement store is fit all constraints and isolation score is not less than the origin.
+func (f *RegionFit) Replace(srcStoreID uint64, dstStore *core.StoreInfo, region *core.RegionInfo) bool {
+	fit := f.getRuleFitByStoreID(srcStoreID)
```

Comment:
Is it the same as the original implementation? Right now it only compares one RuleFit.

Reply:
because other rule fit does not change, the peer store is same.